Makefile

```
@@ -13,111 +15,104 @@ L=[01;30m
 
 npm_config_loglevel="warn"
 
-comandos:
-	@echo ""
-	@echo "${B}Comandos disponibles para ${G}pilas-bloques${N} - ${Y} versión ${VERSION}${N}"
-	@echo ""
-	@echo "  ${Y}Para desarrolladores${N}"
-	@echo ""
-	@echo "    ${G}compilar${N}        Genera los archivos compilados."
-	@echo "    ${G}compilar_live${N}   Compila de forma contínua."
-	@echo "    ${G}compilar_web${N}    Genera la aplicación para la versión web (desde un iframe)."
-	@echo "    ${G}test_travis${N}     Ejecuta las pruebas como esperamos en travis (en paralelo)."
-	@echo ""
-	@echo "    ${L}El comando full es equivalente a realizar estos pasos en orden:${N}"
-	@echo "${L}"
-	@echo ""
-	@echo "  ${Y}Para distribuir${N}"
-	@echo ""
-	@echo "    ${G}version_patch${N}     Genera una versión (0.0.PATCH)."
-	@echo "    ${G}version_minor${N}     Genera una versión (0.MINOR.0)."
-	@echo "    ${G}version_major${N}     Genera una versión (MAJOR.0.0)."
-	@echo ""
-	@echo "    ${L}NOTA: toda versión genera un binario automáticamente en travis, ${N}"
-	@echo "    ${L}      y solamente las versiones minor y major serán distribuidas${N}"
-	@echo "    ${L}      oficialmente. Las versiones patch son internas o de prueba.${N}"
-	@echo ""
-	@echo "    ${G}empaquetar${N}          Genera los binarios de forma local."
-	@echo ""
-	@echo ""
-	@echo ""
-
-
-compilar_ejercicios_pilas: # Para cuando se quiere probar los cambios a ejercicios_pilas SIN releasearlo
-	echo "${G}Compilando ejercicios para Pilas Bloques${N}"
-	@cd ../pilas-bloques-exercises; node_modules/grunt-cli/bin/grunt
-	@cp -rf ../pilas-bloques-exercises/dist node_modules/pilas-bloques-exercises/
-
-watch_ejercicios: 
-	echo "${G}Compilando ejercicios para Pilas Bloques${N}"
-	@cd ../pilas-bloques-exercises; node_modules/grunt-cli/bin/grunt watch
-
-build: 
-	@npm run build
-
-compilar_live:
-	$(EMBER) build --watch
-
-compilar_pilasweb: # Para cuando se quiere probar los cambios a pilasweb SIN releasearlo
+commands:
+	@ echo ""
+	@ echo "${B}Commands available for ${G}pilas-bloques${N} - ${Y} versión ${VERSION}${N}"
+	@ echo ""
+	@ echo "  ${Y}for development:${N}"
+	@ echo ""
+	@ echo "    ${G}build${N}								Build the project."
+	@ echo "    ${G}build_live${N}   						Build the project continuously."
+	@ echo "    ${G}build_pilasweb${N}    					Build Pilas Web dependency."
+	@ echo "    ${G}build_pilas_bloques_exercises${N}		Build Pilas Bloques exercises."
+	@ echo ""
+	@ echo "  ${Y}for deployment:${N}"
```

```suggestion
	@ echo "  ${Y}for distribution:${N}"
```

Deployment is a slightly different thing (when we finally publish, performed via pilas-engine-bloques-website's Makefile)